### DISCUSSION TITLE
Is there an easy way to determine which JS features are available for a given version of @babel/preset-env?

### DISCUSSION THREAD

**Discussion by mattdiamond** [babel/babel#11792] [Q&A]
Buried in the docs for @babel/preset-env is a link to a list of default plugins:

https://github.com/babel/babel/blob/v7.10.4/packages/babel-compat-data/scripts/data/plugin-features.js

Is this the only place this information is stored? It seems pretty important, and yet the documentation only mentions it in passing. Is there another way for a developer to find out which plugins are included with a specific version of preset-env, or is the above file the only resource for this information?

**Comment by JLHwung** [comment]
Per https://babeljs.io/docs/en/babel-preset-env#debug, `debug: true` will output the active plugins given your targets.
```json
{
  "presets": [
    [
      "@babel/preset-env",
      {
        "debug": true,
        "target": { "chrome": 80 }
      }
    ]
  ]
}
```
will output
```
Using targets:
{
  "chrome": "80"
}

Using modules transform: auto

Using plugins:
  syntax-numeric-separator { "chrome":"80" }
  proposal-class-properties { "chrome":"80" }
  proposal-private-methods { "chrome":"80" }
  syntax-nullish-coalescing-operator { "chrome":"80" }
  syntax-optional-chaining { "chrome":"80" }
  syntax-json-strings { "chrome":"80" }
  syntax-optional-catch-binding { "chrome":"80" }
  syntax-async-generators { "chrome":"80" }
  syntax-object-rest-spread { "chrome":"80" }
  transform-modules-commonjs { "chrome":"80" }
  proposal-dynamic-import { "chrome":"80" }
```

You can also query `@babel/compat-data` programmatically
```js
require("@babel/compat-data/plugins")
```
which is a plain JSON like
```jsonc
{
  "proposal-numeric-separator": { "chrome": 75 },
  "proposal-class-properties": { "chrome": 74 },
  // other plugin names
}
```
